### PR TITLE
Posting normal stock movement from entry

### PR DESCRIPTION
**Pull request by DedrickEnc**
This P.R posts normal stock movement from entry to the posting journal through voucher.
The records are first inserted to the voucher table and then inset to the posting journal.

The file ```posting.sql``` is deleted

### CODE REVIEW

**Comment by DedrickEnc** [comment]
@mbayopanda 

Can you review this?

File: server/controllers/stock/index.js [hunk 1]
```diff
@@ -102,16 +102,9 @@ function createStock(req, res, next) {
...
-    db.bid(document.uuid), document.date,
-    req.session.enterprise.id, req.session.project.id,
-    req.session.enterprise.currency_id, document.user,
-  ];
+  const postingParams = [db.bid(document.uuid), 0, req.session.project.id, req.session.enterprise.currency_id];
 
-
-  // writting all records relative to the movement in the posting journal table
-  transaction.addQuery('CALL PostPurchase(?)', [commonInfos]);
+  transaction.addQuery('CALL PostStockMovement(?)', [postingParams]);
```
Comment (mbayopanda) [inline server/controllers/stock/index.js:107]: 👍

**Review by mbayopanda — APPROVED** [review]
It works very well.

LGTM

Could you just rebase this branch with master.

**Pull request merged**